examples/front_ends/responses_api_endpoint/README.md

```
@@ -0,0 +1,208 @@
+curl -X POST http://localhost:8088/v1/responses \
+  -H 'Content-Type: application/json' \
+  -d '{
+    "model": "gpt-4o-mini",
+    "input": "What time is it?",
+    "stream": true
+  }'
+```
+
+**Expected SSE Events:**
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)